site/reference/cli/live/apply/README.md

```diff
@@ -88,18 +88,24 @@ PKG_PATH | -:
 
 --prune-timeout:
   The threshold for how long to wait for all pruned resources to be
-  deleted before giving up. If this flag is not set, kpt live apply will not
-  wait. In most cases, it would also make sense to set the
+  deleted before giving up. If this flag is not set, kpt live apply will wait
+  until interrupted. In most cases, it would also make sense to set the
   --prune-propagation-policy to Foreground when this flag is set.
 
 --reconcile-timeout:
   The threshold for how long to wait for all resources to reconcile before
-  giving up. If this flag is not set, kpt live apply will not wait for
-  resources to reconcile.
+  giving up. If this flag is not set, kpt live apply will wait until
+  interrupted.
 
 --server-side:
   Perform the apply operation server-side rather than client-side.
   Default value is false (client-side).
+
+--status-events:
```

Comment:
is it a boolean ? If yes, we should include the default value (true or false)

if it is boolean, we can probably rename it to `--enable-status-events` or `--{display|show}-status-events` ?

Reply:
I have a PR to change the name to `--show-status-events`: https://github.com/GoogleContainerTools/kpt/pull/2648.

I have updated the docs with the new name and included the default value.